tt_metal/impl/debug/dprint_server.cpp

```diff
@@ -182,9 +195,9 @@ static void PrintTileSlice(ostream& stream, uint8_t* ptr, int hart_id) {
 
 // Writes a magic value at wpos ptr address for dprint buffer for a specific hart/core/chip
 // Used for debug print server startup sequence.
-void WriteInitMagic(int chip_id, const CoreCoord& core, int hart_id, bool enabled) {
+void WriteInitMagic(int chip_id, const CoreCoord& core, int hart_id, bool enabled, bool is_eth_core) {
```

Comment:
we have this api from llrt. 

`inline bool is_ethernet_core(const CoreCoord &core, chip_id_t chip_id)`, it might be better if you try to use this one instead.

Reply:
Thanks for letting me know about this, allowed for some extra cleanup in the latest commit